logger.go

```diff
@@ -0,0 +1,27 @@
+package bigcache
+
+import (
+	"log"
+	"os"
+)
+
+// Logger is invoked when `Config.Verbose=true`
+type Logger interface {
+	Printf(format string, v ...interface{})
+}
+
+var _ Logger = &log.Logger{}
+
+// DefaultLogger returns a `Logger` implementation
+// backed by stdlib's log
+func DefaultLogger() *log.Logger {
```

Comment:
I'm not sure if this should be a function or a package variable. Just like [net.http.DefaultClient](https://golang.org/pkg/net/http/#pkg-variables)

Reply:
tots agree 👍 
Idea was to not rely global state. global vars allow things like:

```
package main

import "net/http"

func main() {
	http.DefaultClient = &http.Client{}
}
```

hence the function route